src/test/java/sorald/ClasspathModeTest.java

```diff
@@ -19,6 +19,9 @@
     @Test
     void resolveClasspathFrom_enablesRepairOfViolation_thatRequiresClasspathToDetect(
             @TempDir File workdir) throws IOException {
+        // use the default security manager
+        System.setSecurityManager(null);
```

Comment:
@algomaster99 Just as a note here, there is no "default security manager", really. Setting it to `null` means there is _no_ security manager, which happens to be the default.

I'd personally also have added a link to the issue detailing why this is necessary. The next person who gets here will have no idea of why we must unset the security manager.

Reply:
> Setting it to null means there is no security manager, which happens to be the default.

Setting it to `null` *indirectly* means that we have the default `SecurityManager` enabled, like you said. So are you saying that it is incorrect to not specify the premise and directly write the conclusion? 

> I'd personally also have added a link to the issue detailing why this is necessary.

I gave a thought about this too. However, I felt that I would have to write that same link at 4 places in the code. Instead, I felt that it would be better if someone just looks up the PR because I have linked the issues properly in the  description.

Reply:
> Setting it to null indirectly means that we have the default SecurityManager enabled, like you said. So are you saying that it is incorrect to not specify the premise and directly write the conclusion?

Well, it's a little bit misleading, because there is no default security manager. The default is to not have a security manager. Saying "use the default security manager" implies that there actually is one. But `null` is the absence of a value, and `null` is not a security manager.

> Instead, I felt that it would be better if someone just looks up the PR because I have linked the issues properly in the description.

That's great until someone modifies that line and (perhaps changes indentation, moves it, etc) and `git blame` gives you the wrong answer, and you have to start a bisect to find where the line originated. Perhaps not even in the same file. Sooo, I'd definitely have duplicated that link.

Reply:
For context, a typical Java application does not run with a security manager: https://docs.oracle.com/javase/tutorial/essential/environment/security.html

Reply:
Right! I will update the comments.

Reply:
Eh, too much effort imo :). I mean we want to fix this in Spoon, so this should all be removed after that.

Reply:
> I mean we want to fix this in Spoon

Yeah, we won't need this then. :)
I will just create an issue so that at least we don't forget about it.